samples/dotnet/github-skills/GitHubSkill.cs

```diff
@@ -176,6 +163,25 @@ public async Task SummarizeRepositoryAsync(string source, SKContext context)
         }
     }
 
+    private async Task DownloadToFileAsync(string uri, IDictionary<string, string> headers, string filePath, CancellationToken cancellationToken)
```

Comment:
@adrianwyatt is this a pattern that we want to follow? (Reverting the single-responsibility principle, and duplicating code  instead of reusing it from webskill), or is this only temporary? I am ok, with either decision, just want to make sure it is intentional.

Reply:
As a sample project we are not worried about duplication of code at this point.